boards/xtensa/esp32/esp32-core/src/esp32_bringup.c

```diff
@@ -143,7 +143,7 @@ int esp32_bringup(void)
 
 #endif
 
-#if defined(CONFIG_ESP32_SPIRAM)
+#if defined(CONFIG_ESP32_SPIRAM) && defined(CONFIG_SPIRAM_BANKSWITCH_ENABLE)
```

Comment:
CONFIG_SPIRAM_BANKSWITCH_ENABLE or CONFIG_ESP32_SPIRAM_BANKSWITCH_ENABLE?
In _arch/xtensa/src/esp32/Kconfig_, this configure option changed to ESP32_SPIRAM_BANKSWITCH_ENABLE

Please kindly check.

Thanks

Reply:
Thank you! I was going to replace CONFIG_SPIRAM_BANKSWITCH_ENABLE with CONFIG_ESP32_SPIRAM_BANKSWITCH_ENABLE but I regret on it and forgot to remove it from a place in the Kconfig.